src/foam/u2/detail/TabbedDetailView.js

```diff
@@ -78,13 +78,16 @@ foam.CLASS({
                         code: s.title
                       }) :
                       s.title$;
+
+                    var tab = foam.core.SimpleSlot.create();
                     this
-                      .start(self.Tab, { label$: title$ || self.defaultSectionLabel, selected: self.memento.paramsObj.selectedTab && self.memento.paramsObj.selectedTab === s.title })
+                      .start(self.Tab, { label$: title$ || self.defaultSectionLabel, selected: self.memento.paramsObj.selectedTab && self.memento.paramsObj.selectedTab === s.title }, tab)
                         .call(function() {
                           this.tag(self.SectionView, {
                             data$: self.data$,
                             section: s,
-                            showTitle: false
+                            showTitle: false,
+                            loadLatch$: tab.value.selected$.map(function(s) {return ! s;})
```

Comment:
The loadLatch should just be a latch version of selected. It turns true but then never turns back. So, just add 'selected' to SectionView and on the postSet of it, set loadLatch to true if the new value is true. 
Then change line 138 to just if ( loadLatch )